src/data/quality-assurance-quiz.ts

```diff
@@ -182,7 +182,61 @@ const qualityAssuranceQuiz = [
     Explanation:
       "A good QA engineer should understand the full life cycle of software products in order to effectively plan and test them. There are 7 phases of the Software Development Life Cycle",
     Link: "https://www.freecodecamp.org/news/how-to-become-a-quality-assurance-engineer-qa-engineer-career-guide/"
-  }
+  },
+  {
+      Question: "What is the primary purpose of a test plan in software quality assurance?",
+      Answer: "A test plan outlines the scope, approach, resources, and schedule for testing activities. Its primary purpose is to guide the testing process and ensure that all aspects of testing are systematically covered.",
+      Distractor1: "A test plan is used to document defects found during testing.",
+      Distractor2: "A test plan specifies the code changes needed to fix defects.",
+      Distractor3: "A test plan is only created after testing is complete.",
+      Explanation: "Test plans are essential for planning, managing, and executing testing efforts in a structured manner. They help teams achieve comprehensive test coverage and meet quality goals.",
+      Link: "https://www.guru99.com/test-plan.html"
```

Comment:
we only accept links from freeCodeCamp news publication, freeCodeCamp YouTube channel, or wikipedia
Please change this link